game/util.cc

```diff
@@ -32,3 +36,112 @@ std::string format(std::chrono::seconds const& unixtime, std::string const& form
 
 	return stream.str();
 }
+
+
+std::string toLower(std::string const& s) {
+	//return boost::algorithm::to_lower_copy(s);
+	auto result = s;
+
+	std::for_each(result.begin(), result.end(), [](auto& c){ c = static_cast<char>(std::tolower(c));});
+
+	return result;
+}
+
+std::string toUpper(std::string const& s) {
+	//return boost::algorithm::to_upper_copy(s);
+	auto result = s;
+
+	std::for_each(result.begin(), result.end(), [](auto& c){ c = static_cast<char>(std::toupper(c));});
+
+	return result;
+}
+
+std::string erase(std::string const& s, std::string const& toErase) {
+	auto result = s;
+
+	return erase(result, toErase);
+}
+
+std::string erase(std::string& s, std::string const& toErase) {
+	if(toErase.empty())
+		return s;
+
+	for(auto position = s.find(toErase); position < s.length(); position = s.find(toErase, position))        s.erase(position, toErase.length());
+
+	return s;
+}
+
+std::string replace(std::string const& s, char from, char to) {
+	auto result = s;
+
+	return replace(result, from, to);
+}
+
+std::string& replace(std::string& s, char from, char to) {
```

Comment:
Might be better to add a formatter whether to ignore case or not. Now this won't replace things: `replace('a', 'A')` and it's not clear why.

Reply:
Don't know if I changed it as you expected. Please check that.